crud_app.py

```
@@ -0,0 +1,56 @@
+import psycopg2
+
+try:
+    conn = psycopg2.connect("dbname=amardeep user=amardeep password='Vtu.topper004'")
+
+
+except:
```

Exception is not handled.Print is not needed in production level code.